extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -381,23 +405,27 @@ void ConsumeWindowsEventLog::onTrigger(const std::shared_ptr<core::ProcessContex
     }
   }
 
+  logger_->log_trace("Finish enumerating events.");
```

Comment:
Shouldn't we also yield here, when there are no more events left?

Reply:
This can mean that the processor gathered a bunch of events (and those events has been generated since the last ontrigger call), so I don't think waiting  for a longer time helps in this case.
Yield might be  considered in case the ontrigger call generated 0 new events.